pupil_src/shared_modules/head_pose_tracker/controller/online_controller.py

```diff
@@ -60,6 +62,17 @@ def _calculate_current_pose(self, frame):
             self._camera_intrinsics,
         )
 
+    def _create_position_events(self, current_pose):
```

Comment:
The parameter `current_pose` will never be used with something else than `self.__localization_storage.current_pose`.
I'd suggest you remove the parameter and just do this below:
```python
position.update(self._localization_storage.current_pose)
```